modular_skyrat/modules/brigoff/code/brigoff.dm

```diff
@@ -54,11 +54,11 @@
 	shoes = /obj/item/clothing/shoes/laceup
 	ears = /obj/item/radio/headset/headset_sec
 	glasses = /obj/item/clothing/glasses/sunglasses
-	backpack_contents = list(/obj/item/melee/baton, /obj/item/restraints/handcuffs = 2, /obj/item/gun/energy/taser/armadyne)
```

Comment:
If you're gonna remove their taser, give them a disabler.

Reply:
Sure.